src/main/java/com/spm/vasylyshyn/model/User.java

```diff
@@ -41,18 +47,28 @@ public class User implements UserDetails {
     @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.PERSIST, mappedBy = "owner")
     private List<Device> deviceList;
 
-    @JsonFormat(pattern = "yyyy-mm-dd HH:mm:ss")
-    @Column(updatable = false)
-    private LocalDateTime createdDate;
+    private String imageUrl;
+
```

Comment:
```suggestion
```
I guess we don't use double line breaks between code lines unless for these cases:

Two blank lines should always be used in the following circumstances:

Between sections of a source file
Between class and interface definitions
One blank line should always be used in the following circumstances:
Between methods
Between the local variables in a method and its first statement
Before a block[ (see section 5.1.1)](https://www.cs.toronto.edu/km/istar/OME3/codeconv/CodeConventions.doc4.html#680) or single-line[ (see section 5.1.2)](https://www.cs.toronto.edu/km/istar/OME3/codeconv/CodeConventions.doc4.html#341) comment
Between logical sections inside a method to improve readability

Reply:
Ok